reduce.js

```diff
@@ -8,3 +8,8 @@ var numbers = [1, 56, 2, 4, 1, 99, 3, 5];
 // via multiplication
 
 
+const reducer = (num1,num2) => {
+  return num1*num2
+}
+
+console.log(numbers.reduce(reducer))
```

Comment:
nice work here! it's typical to name the accumulator paramater `accum`, in order to keep track of it as it's an important part of how this method works. It doesn't matter as far as functionality, but it is useful as far as human readability. 

Also, a question for you to keep in mind: what does `reduce()` treat as the accumulator if you don't specify one?